### ISSUE TITLE
Make use of the LHCb glossary

### ISSUE THREAD

**Issue by eduardo-rodrigues** [lhcb/starterkit-lessons#179]
Hi all,

It would be great if the starterkit material start making use of the glossary recently prepared, see https://github.com/lhcb/glossary. This way, relevant definitions can all be collected in one place and referred to in the various training and documentation material sets.

Many thanks.

**Comment by chrisburr** [comment]
I'll leave this open as I think it would be nice to actually use the glossary to link words to explanations.

**Comment by eduardo-rodrigues** [comment]
Precisely. Having a link to the glossary is just a starting point. Key words ought to be linked to the glossary :-).
